static/js/libs/arrays.test.js

```diff
@@ -45,6 +45,18 @@ describe("arraysEqual", () => {
   it("should fail if an array of objects differ", () => {
     expect(arraysEqual([{ test: "test" }], [{ test: "test2" }])).toEqual(false);
   });
+
+  it("should fail if an array is modified", () => {
```

Comment:
One more test case I would add is to make sure `arraysEqual` compares 2 identical arrays longer that 1 element.
Our current test compare same 1 element arrays. The only test featuring 2 item array is the one that compares arrays with different lengths.
Current tests would still pass if our implementation would only compare first element of the array.

We should add tests for cases like:

`expect(arraysEqual(["a", "b", "c"], ["a", "b", "c"])).toEqual(true);`
`expect(arraysEqual(["a", "b", "c"], ["a", "d", "c"])).toEqual(false);`